install.sh

```diff
@@ -543,6 +543,17 @@ EOABORT
 )"
 fi
 
+# Multi-user support on macOS
+directories=("$HOMEBREW_REPOSITORY" "$HOMEBREW_CACHE")
```

Comment:
I don't think it makes sense to do this for `HOMEBREW_CACHE` because it's inside the `HOME` of a specific user (and will change depending on which user runs it)